cmd/applications-rp/radius-self-hosted.yaml

```diff
@@ -8,9 +8,9 @@
 # - Disables metrics and profiler
 #
 environment:
-  name: Dev
+  name: self-hosted
```

Comment:
just for my own understanding, is there a reason why Dev starts with a capital letter and self-hosted not?

Reply:
No idea. These names are just descriptive, they have no behavior associated.